src/Symfony/Bridge/PhpUnit/Legacy/CommandForV5.php

```diff
@@ -23,9 +23,7 @@ class CommandForV5 extends \PHPUnit_TextUI_Command
      */
     protected function createRunner()
     {
-        $listener = new SymfonyTestsListenerForV5();
-
-        $this->arguments['listeners'] = isset($this->arguments['listeners']) ? $this->arguments['listeners'] : array();
+        $this->arguments['listeners'] = isset($this->arguments['listeners']) ? $this->arguments['listeners'] : [];
```

Comment:
This needs to be reverted. The bridge is compatible with PHP 5.3.

Reply:
@xabbuh fabbot takes issue with this, is it misconfigured?

Reply:
fabbot use the same configuration for the entire repository. the PHPUnitBridge don't have the same requirements than other components